group_vars/geniza/vars.yml

```diff
@@ -46,3 +46,12 @@ solr_configset: geniza
 # github contexts required for deploy
 deploy_contexts:
   - "Python unit tests"
+
+# configure scripts to run as cron jobs
+crontab:
+
+  - name: "{{ app_name }} reindex"
+    minute: 3
```

Comment:
just for clarity, is this 12:03 am daily? a comment might be handy to indicate what real time it corresponds to

Reply:
yes, good point.

Reply:
Actually, I copied from PEMM and just remembered/figured out that it's indexing every hour at 3 minutes past the hour! I'll specify an hour. Also, I checked the logs and discovered that I need to specify `--no-progress` 😆